server/mailet/mock-smtp-server/src/test/java/org/apache/james/mock/smtp/server/HTTPConfigurationServerTest.java

```diff
@@ -225,6 +226,27 @@ void getShouldReturnMultipleEmails() throws Exception {
                 .isEqualTo(JSON_MAILS_LIST);
         }
 
+        @Test
+        void getShouldNotReturnClearedEmails() {
+            mailRepository.store(mail1);
+            mailRepository.store(mail2);
+
+            with().delete();
```

Comment:
CR

Reply:
If you wish.

I believe here it does not hurt readability at all.

I believe to some extend cosmetic display choice could be left to the author, no?